mmv1/products/firestore/Index.yaml

```diff
@@ -152,6 +158,12 @@ properties:
       definition reach or traverse an array, except via an explicit array
       index. Violations will result in errors. Note this field only applies to
       indexes with MONGODB_COMPATIBLE_API ApiScope.
+  - name: 'unique'
+    type: Boolean
+    default_value: false
+    description:
+      Optional. Whether it is an unique index. Unique index ensures all values for the
+      indexed field(s) are unique across documents.
```

Comment:
"unique across documents" means "unique across all documents indexed by this index as defined by its collection and query scope", right?